now.json

```diff
@@ -0,0 +1,12 @@
+{
+  "version": 2,
+  "public": true,
+  "scope": "aragon",
+  "alias": "nightly-rinkeby.aragon.org",
+  "builds": [
+    { "src": "package.json", "use": "@now/static-build", "config": { "distDir": "build/rinkeby" } }
+  ],
+  "github": {
+    "silent": true
```

Comment:
Is it so that we can add the links ourselves in the PR descriptions?

Reply:
Mostly so the bot doesn't comment on every push with the deployment.

There's already quite a few places where you can get the link (the "view deployment"s on the right, the "deployment" message before the merge box, and then in the checks themselves).